cli/daemon/engine/trace/trace.go

```diff
@@ -78,14 +80,16 @@ func (st *Store) List(appID string) []*TraceMeta {
 	return tr
 }
 
-func Parse(traceID ID, data []byte, proc *run.Proc) ([]*tracepb.Request, error) {
+func Parse(log *zerolog.Logger, traceID ID, data []byte, version trace.Version, symTable SymTable) ([]*tracepb.Request, error) {
```

Comment:
This is a merge of the two copies of this code.

Basically keeping the extra stuff the local dev does, while allowing the platform to use it and have backwards compatibility with older stored traces